README.md

```diff
@@ -969,37 +969,65 @@ cloudos job workdir --status --profile my_profile --job-id "12345678910" --verbo
 
 #### List Jobs
 
-You can get a summary of workspace jobs in two different formats:
+View your workspace jobs in a clean, formatted table directly in your terminal. By default, jobs are displayed as a rich table with clickable job IDs and visual status indicators.
```

Comment:
```suggestion
View your workspace jobs in a clean, formatted table directly in your terminal. By default, jobs are displayed as a rich table with job IDs and visual status indicators.
```
They are not always clickable, only in terminals with some theme enhancements or in IA sessions, but not in vanilla terminals.